### PR TITLE
Add few changes to get it run for current maintainer

### PR DESCRIPTION
**Pull request by neurogoo**

### CODE REVIEW

File: src/Site.hs [hunk 1]
```diff
@@ -87,7 +89,7 @@ main = shakeArgs shakeOptions $ do
 
         let relativeXzs = [ "./" ++ f ++ ".xz" | f <- files ]
 
-        Stdout sums <- cmd ("sha256sum" : relativeXzs) (Cwd "site/files")
```
Comment (phadej) [inline src/Site.hs:90]: you might add a note, that this is `Linux -> macOS` change

File: src/Site.hs [hunk 2]
```diff
@@ -87,7 +89,7 @@ main = shakeArgs shakeOptions $ do
...
+        Stdout sums <- cmd (["shasum", "-a", "256"] <> relativeXzs) (Cwd "site/files")
```
Comment (phadej) [inline src/Site.hs]: and `++` will make it work on CI

**Review by phadej — COMMENTED** [review]
Thanks!

File: src/Site.hs (hunk 1 above)
Comment (neurogoo) [reply src/Site.hs:90]: This was yes, but it should also work on Linux? At least Ubuntu does have shasum exe

Comment (phadej) [reply src/Site.hs:90]: TIL! though it's not an exe, it's a perl script :) :+1:

**Pull request merged**